examples/hpc/dla-driver/src/utils.rs

```diff
@@ -78,7 +76,6 @@ pub fn get_banks_for_layer(
     input_size: usize,
     kernels_size: usize,
     output_size: usize,
-    bias_size: Option<usize>,
```

Comment:
Okay, now I understand what you meant in the previous PR. This makes way more sense.